newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -208,6 +221,41 @@ export default class PropertiesEditor extends React.Component<Props, {||}> {
           disabled={field.disabled}
         />
       );
+    } else if (field.valueType === 'color') {
+      const { setValue } = field;
+      return (
+        <ColorField
+          key={field.name}
+          id={field.name}
+          floatingLabelText={getFieldLabel(this.props.instances, field)}
+          disableAlpha
+          fullWidth
+          color={getFieldValue(this.props.instances, field)}
```

Comment:
Could we transform here the result of getFieldValue to a RGBColor object? I would rather have this transformation here rather than make the poor ColorField more complex by being able to handle a string.

(this is arguable, but as onChangeComplete is the output is returning a color, which is a `RGBColor`, it's a good idea to stay "symetric" and have the input, `color` to also be a `RGBColor`).
(we can still expose the method `getRgbBackground` as a tool method that you export, but in this case rename it "getRGBColorFromHex")

Reply:
Ah yes you're right. I will move it out :)